frontend/src/core/user/components/UserMenu.js

```diff
@@ -0,0 +1,208 @@
+                        }
+                    >
+                        <a href={ `/${parameters.locale}/${parameters.project}/${parameters.locale}.${parameters.project}.tmx` }>
+                            { '<glyph></glyph>Download Translation Memory' }
+                        </a>
+                    </Localized>
+                </li>
+
+                <li className="horizontal-separator"></li>
+
+                <li>
+                    <Localized
+                        id="user-UserMenu--top-contributors"
+                        glyph={
+                            <i className="fa fa-trophy fa-fw"></i>
+                        }
+                    >
+                        <a href="/contributors/">
+                            { '<glyph></glyph>Top Contributors' }
+                        </a>
+                    </Localized>
+                </li>
+
+                <li>
+                    <Localized
+                        id="user-UserMenu--machinery"
+                        glyph={
+                            <i className="fa fa-search fa-fw"></i>
+                        }
+                    >
+                        <a href="/machinery/">
+                            { '<glyph></glyph>Machinery' }
+                        </a>
+                    </Localized>
+                </li>
+
+                <li>
+                    <Localized
+                        id="user-UserMenu--terms"
+                        glyph={
+                            <i className="fa fa-gavel fa-fw"></i>
+                        }
+                    >
+                        <a href="/terms/">
+                            { '<glyph></glyph>Terms of Use' }
+                        </a>
+                    </Localized>
+                </li>
+
+                <li>
+                    <Localized
+                        id="user-UserMenu--help"
+                        glyph={
+                            <i className="fa fa-life-ring fa-fw"></i>
+                        }
+                    >
+                        <a href="https://mozilla-l10n.github.io/localizer-documentation/tools/pontoon/">
+                            { '<glyph></glyph>Help' }
+                        </a>
+                    </Localized>
+                </li>
+
+                { !user.isAuthenticated ? null :
+                <li className="horizontal-separator"></li>
+                }
+
+                { !user.isAdmin ? null :
+                <li>
+                    <Localized
+                        id="user-UserMenu--admin"
+                        glyph={
+                            <i className="fa fa-wrench fa-fw"></i>
+                        }
+                    >
+                        <a href="/admin/">
+                            { '<glyph></glyph>Admin' }
+                        </a>
+                    </Localized>
+                </li>
+                }
+
+                { !user.isAdmin ? null :
```

Comment:
Same here, why not have those two be part of the same block since the conditions are the same?